### DISCUSSION TITLE
Dark Theme for Ananke

### DISCUSSION THREAD

**Discussion by mirontoli** [gohugo-ananke/ananke#668] [2 - Q&A / Support] [answered]
Hi, I would like to have Dark Theme or (even better) that the site built with Hugo and Ananke should follow the user preferred theme. Is there a way to have dark and light themes?

**Comment by budparr** [comment, marked as answer]
Hi @mirontoli We are in maintenance mode with this theme and do not plan any new features. Thank you.
